geniza/entities/tests/test_entities_models.py

```diff
@@ -294,6 +294,11 @@ def test_merge_with_log_entries(self):
             in moved_log.change_message
         )
 
+    def test_get_absolute_url(self):
+        # should get person page url in user's language by pk
+        person = Person.objects.create()
+        assert person.get_absolute_url() == "/en/people/%s/" % person.pk
```

Comment:
Can you add a slug field for use in urls? I have a strong preference for not using database id numbers for public site urls.  We can discuss more if needed/helpful - I haven't done on a multilingual site before, so you might want multiple slugs. We have logic on other projects to track past slugs when it's possible they could change, so you can resolve if something changes (both for datasets and for webapp urls).

Reply:
I recognize that we're using database ids for PGPIDs but those are an exception to me since we're preserving / continuing an existing identifier system that existed previous to PGPv4 and also because there's not a reliable way we could generate short, brief, stable slug-type identifiers for documents.